arcaflow_plugin_sysbench/sysbench_schema.py

```diff
@@ -696,16 +692,14 @@ class WorkloadResultsCpu:
         SysbenchCpuOutputParams,
         schema.name("Sysbench Cpu Output Parameters"),
         schema.description(
-            "Ouptut parameters for a successful sysbench cpu workload"
-            " execution"
+            "Ouptut parameters for a successful sysbench cpu workload" " execution"
```

Comment:
Can be merged into one string

Reply:
Those must have been linting quirks. I'll clean them up.